oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreBranch.java

```diff
@@ -130,6 +130,15 @@ public NodeState merge(@NotNull CommitHook hook, @NotNull CommitInfo info)
         return merge0(hook, info, true);
     }
 
+    /**
+     * For test purposes only!
+     * <p>
+     * Forces the branch to persist the changes to the underlying store.
+     */
+    void persist() {
+        branchState.persist();
+    }
+
```

Comment:
Can you please move this method further down and place it between `getMergeLock()` and `merge0()`? This matches the location in trunk and reduces potential conflicts in the future when further changes a ported back.